plugin.video.catchuptvandmore/resources/lib/channels/be/rtlplaybe.py

```diff
@@ -107,11 +108,28 @@
 DESIRED_QUALITY = Script.setting['quality']
 
 
-def replay_entry(plugin, item_id, **kwargs):
-    """
-    First executed function after replay_bridge
-    """
-    return list_categories(plugin, item_id)
+@Route.register
+def rtlplay_root(plugin, **kwargs):
+
+    # (item_id, label, thumb, fanart)
+    channels = [
+        ('rtl_tvi', 'RTL TVI', 'rtltvi.png', 'rtltvi_fanart.jpg'),
+        ('club_rtl', 'CLUB RTL', 'clubrtl.png', 'clubrtl_fanart.jpg'),
+        ('plug_rtl', 'PLUG RTL', 'plugrtl.png', 'plugrtl_fanart.jpg'),
+        ('rtl_info', 'RTL INFO', 'rtlinfo.png', 'rtlinfo_fanart.jpg'),
+        ('rtl_sport', 'RTL Sport', 'rtlsport.png', 'rtlsport_fanart.jpg'),
+        ('bel_rtl', 'BEL RTL', 'belrtl.png', 'belrtl_fanart.jpg'),
+        ('contact', 'Contact', 'contact.png', 'contact_fanart.jpg')
```

Comment:
is contact a channel or a way of contact? if the later where does the message go to?

Reply:
Yes, Contact is a Belgium radio from the RTL group (https://www.rtlplay.be/contact). Good Radio by the way 😛